zipline-gradle-plugin/src/test/kotlin/app/cash/zipline/gradle/ZiplineDownloaderTest.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.cash.zipline.gradle
+
+import java.io.File
+import org.junit.Test
+
+class ZiplineDownloaderTest {
+  private val ziplineDownloader = ZiplineDownloader()
+
+  @Test
+  fun `integration test to load from real url`() {
```

Comment:
Is this doing external HTTP calls? I'm not comfortable with such a test being enabled by default. My suite shouldn't fail just because I'm testing on a plane.

Reply:
Maybe MockWebServer??